### ISSUE TITLE
Normalization parameters used for the training set

### ISSUE THREAD

**Issue by 464hee** [lunit-io/benchmark-ssl-pathology#3]
I would like to know in what way dino_vit_small_patch16_ep200.torch goes through the normalization of the training images, because in practice we often put the test images through the same normalization process as the training set, e.g. 

transforms_beit3 = transforms.Compose([
    transforms.Resize((224,224), interpolation=3), transforms.
    transforms.ToTensor(),
    transforms.Normalize(mean= (0.5, 0.5, 0.5), std= (0.5, 0.5, 0.5))
        ])

trnsfrms_CTransPath_Brow_iBOTViT = transforms.Compose(
    [
    transforms.Resize((224,224)),
    transforms.ToTensor(),
    transforms.Normalize(mean = (0.485, 0.456, 0.406), std = (0.229, 0.224, 0.225))
    ]
)

What way are the weights you trained referencing to perform normalization? Or not standardized
@chrisavl @deminoth @swook @apaek @ghilbut

**Comment by swook** [comment]
Thanks for pointing out this omission. It was not intentional.

Please find the stats used for image standardization at: https://github.com/lunit-io/benchmark-ssl-pathology/releases/tag/pretrained-weights

<blockquote><img src="https://opengraph.githubassets.com/e8a6941d7ecac9d2f2f044dc56c553799f8a1ec0b3402caba4dedac7c38126ab/lunit-io/benchmark-ssl-pathology/releases/tag/pretrained-weights" width="48" align="right"><div><img src="https://github.githubassets.com/favicons/favicon.svg" height="14"> GitHub</div><div><strong><a href="/lunit-io/benchmark-ssl-pathology/releases/tag/pretrained-weights">Release Self-supervised pre-trained weights on TCGA · lunit-io/benchmark-ssl-pathology</a></strong></div><div>Benchmarking Self-Supervised Learning on Diverse Pathology Datasets
We execute the largest-scale study of SSL pre-training on pathology image data. Our study is conducted using 4 representative SSL...</div></blockquote>

**Comment by 464hee** [comment]
Thank you for your replies.
